qemu/tests/win_heavyload.py

```diff
@@ -58,6 +58,19 @@ def add_option(cmd, key, val):
             cmd += " /%s %s " % (key, val)
         return cmd
 
+    def cleanup_temp(session):
+        """
+        del temp file that created by heavyload in VM
+
+        :param session: VM session
+        """
+        check_temp_cmd = 'dir %temp% %tmp% /s /b | findstr jam*'
+        s, o = session.cmd_status_output(check_temp_cmd)
+        if not s:
```

Comment:
what if `s=1`, it means no temp file?  Does this meaning there could be an error?

Reply:
No, it is not an error.
When heavyload auto exit, temp file will be deleted automatically. But when heavyload  can not exit and be killed with "stop_cmd", temp file can not be deleted, we need delete it here.

Reply:
got it, thanks